pkg/cmd/cmd.go

```diff
@@ -63,30 +106,27 @@ func NewPatchCmd() *cobra.Command {
 		},
 	}
 	flags := patchCmd.Flags()
+	flags.StringVar(&ua.configFile, "config", "", "Path to a bulk patch YAML config file. If used, --image and --report are ignored.")
 	flags.StringVarP(&ua.appImage, "image", "i", "", "Application image name and tag to patch")
 	flags.StringVarP(&ua.reportFile, "report", "r", "", "Vulnerability report file or directory path")
 	flags.StringVarP(&ua.patchedTag, "tag", "t", "", "Tag for the patched image")
 	flags.StringVarP(&ua.suffix, "tag-suffix", "", "patched", "Suffix for the patched image (if no explicit --tag provided)")
 	flags.StringVarP(&ua.workingFolder, "working-folder", "w", "", "Working folder, defaults to system temp folder")
 	flags.StringVarP(&ua.bkOpts.Addr, "addr", "a", "", "Address of buildkitd service, defaults to local docker daemon with fallback to "+buildkit.DefaultAddr)
-	flags.StringVarP(&ua.bkOpts.CACertPath, "cacert", "", "", "Absolute path to buildkitd CA certificate")
-	flags.StringVarP(&ua.bkOpts.CertPath, "cert", "", "", "Absolute path to buildkit client certificate")
-	flags.StringVarP(&ua.bkOpts.KeyPath, "key", "", "", "Absolute path to buildkit client key")
-	flags.DurationVar(&ua.timeout, "timeout", 5*time.Minute, "Timeout for the operation, defaults to '5m'")
-	flags.StringVarP(&ua.scanner, "scanner", "s", "trivy", "Scanner used to generate the report, defaults to 'trivy'")
-	flags.BoolVar(&ua.ignoreError, "ignore-errors", false, "Ignore errors and continue patching (for single-platform: continue with other packages; for multi-platform: continue with other platforms)")
-	flags.StringVarP(&ua.format, "format", "f", "openvex", "Output format, defaults to 'openvex'")
-	flags.StringVarP(&ua.output, "output", "o", "", "Output file path")
-	flags.BoolVarP(&ua.push, "push", "p", false, "Push patched image to destination registry")
+	flags.StringVar(&ua.bkOpts.CACertPath, "cacert", "", "Absolute path to buildkitd CA certificate")
+	flags.StringVar(&ua.bkOpts.CertPath, "cert", "", "Absolute path to buildkitd client certificate")
+	flags.StringVar(&ua.bkOpts.KeyPath, "key", "", "Absolute path to buildkitd client key")
+	flags.DurationVar(&ua.timeout, "timeout", 15*time.Minute, "Timeout for the entire operation, defaults to '15m'")
+	flags.StringVarP(&ua.scanner, "scanner", "s", "trivy", "Scanner used to generate reports")
+	flags.BoolVar(&ua.ignoreError, "ignore-errors", false, "Ignore errors during a patch operation and continue with other tasks")
```

Comment:
I think we should leave in the explanation for how ignore errors works for patching multiplatform images